Wikipedia/Code/HistoryViewController.swift

```diff
@@ -165,7 +165,26 @@ class HistoryViewController: ArticleFetchedResultsViewController, WMFNavigationB
             }
         }
     }
-    
+
+    override func previewingViewController(for indexPath: IndexPath, at location: CGPoint) -> UIViewController? {
+        guard let vc = super.previewingViewController(for: indexPath, at: location) else {
+            return nil
+        }
+
+        guard let articleVC = (vc as? ArticleViewController) else {
+            return nil
+        }
+
+        guard let articleURL = articleURL(at: indexPath) else {
+            return nil
+        }
+
+        articleVC.articleViewSource = .history
+        articleVC.articlePreviewingDelegate = self
+        articleVC.wmf_addPeekableChildViewController(for: articleURL, dataStore: dataStore, theme: theme)
```

Comment:
I think you should be able to delete these two lines, the superclass call on 170 above should have already set these up.